source/imaer-gml/src/main/java/nl/overheid/aerius/importer/ImaerImporter.java

```diff
@@ -150,10 +150,9 @@ public void importStream(final InputStream inputStream, final Set<ImportOption>
   private static ScenarioSituation addSituationProperties(final GMLReader reader, final ImportParcel result) {
     final ScenarioSituation situation = result.getSituation();
     situation.setName(reader.getName());
-    final SituationType type = reader.getSituationType();
-    situation.setType(type);
+    situation.setType(reader.getSituationType());
     situation.setReference(reader.metaDataReader().readReference());
-    if (type == SituationType.NETTING) {
+    if (situation.getType() == SituationType.OFF_SITE_REDUCTION) {
```

Comment:
Bit curious about this one, why was the local variable removed? Setters/getters normally don't affect performance I guess so it shouldn't matter, but wonder if there's another reason to do it like this?

Reply:
not really, this change was left behind after some changes were made and partially reverted in this file. I've restored the local variable.